modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -20010,6 +20010,7 @@ this-message-does-not-have-file-attachments-in-the-recycle-bin=This message does
 this-name-is-already-in-use-try-another-one=This name is already in use. Try another one.
 this-number-refers-to-the-total-number-of-reads-since-the-content-was-published=This number refers to the total number of reads since the content was published.
 this-number-refers-to-the-total-number-of-views-since-the-content-was-published=This number refers to the total number of views since the content was published.
+this-object-includes-translatable-fields=This object includes translatable fields. Actions always use the entry's default language.
```

Comment:
Hey @DavysonMelo, can you change it to this key? Please, run again the `gw buildLang`.

this-object-includes-translatable-fields.-actions-always-use-the-object-entry's-default-language=This object includes translatable fields. Actions always use the object entry's default language.

Reply:
remember to update the UI with the new key please 👍

Reply:
If I may add a suggestion, @DavysonMelo, there's no need to create a new commit just to update this! you can change the key, run the buildLang again and squash/fixup these changes into your previously made commits related to key creation and the buildLang, since you're just updating it.